### PR TITLE
Group #7 added translation docs

### PR DESCRIPTION
**Pull request by BestArya**
This article introduces the definition of goroutine and common tools used to test it, which is very helpful for beginners in golang to avoid such common pitfalls. In particular, the article describe goroutine as a 'groundhog', while humorous, it makes the whole concept much easier to understand, even for non-professionals.

### CODE REVIEW

**Comment by zl117** [comment]
comments from group 1:
  This is a good example overall, but I think this translation can be improved.
  "There are many groundhogs at your disposal" in the sentence 11, I know that this is from the Chinese version of this guideline, and this is a perfect version of direct translation. But according to my understanding, English speakers can be easily misguided by some of the phrases that we use, and also to prove my point, I have searched through the Internet, found 0 use of a phrase like this, not even to use a groundhog to do anything. I am aware of the fact that this is used as a metaphor, it will be better if we quote on this or give out the definition right after this sentence. 
  I am asking for advice from a native speaker. Please let me know if I am wrong, thank you!

**Comment by BestArya** [comment]
Solved issue #24

File: blog-en/Testing of Goroutine Leak in PouchContainer.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+## <a name="pobkqe"></a>0. Introduction
+
+[PouchContainer](https://github.com/alibaba/pouch) is an open-source project created by Alibaba Group. It provides strong isolation and high portability, which can help enterprises quickly containerize their stock business and improve the utilization of internal physical resources. 
+
+PouchContainer is also a golang project in which goroutine has been widely used to implement management of various modules including containers, images, and logs. goroutine is an user-mode 'thread' supported by golang at the language level. This built-in feature can help developers build high-concurrency services at a rapid pace.
```
Comment (MaoYiming0725) [inline blog-en/Testing of Goroutine Leak in PouchContainer.md]: line 5: goroutine is an user-mode 'thread' --> Goroutine is an user-mode 'thread'

**Comment by allencloud** [comment]
LGTM

**Pull request merged**